### PR TITLE
Enh/generic droplet2phot

### PR DESCRIPTION
**Pull request by silkenelson**
Have the photonizing code run on the output of the standard droplet instead of the python version. Will thus work with floating point values and also titles detectors (to be tested).

### CODE REVIEW

**Comment by vespos** [comment]
I will go over that PR next week. If you have the chance, do you think you can add a brief description of what each new function does in the docstring (the new ones in droplet2Photon in particular)? 
Although I am pretty familiar with how this all should work, that would help me follow the logic of the code.

File: smalldata_tools/ana_funcs/droplet2Photons.py [hunk 1]
```diff
@@ -8,71 +8,256 @@
...
+            j = pixones[ppos[drop]:ppos[drop+1],1]
+            adus = pixonesadu[ppos[drop]:ppos[drop+1]]
+            maxPixAdu.append(np.nanmax(adus))
+            ##they need to be neighbors, this is not required here....
+            #twoPixAdu.append((np.sort(adus)[-1*(min(2, len(adus))):]).sum())
+            piximg1=sparse.coo_matrix( (adus, (i, j)) ).todense()
+            #find maximum, find maximum of footprint_nbradd neighbor, maximum.
+            nbrpix=filters.maximum_filter(piximg1, mode='constant', footprint=self._footprintnbr).flatten()[np.argmax(piximg)]
+            twonbrPixAdu.append(maxPixAdu[-1]+nbrpix)
+            return maxPixAdu, twonbrPixAdu
```
Comment (vespos) [inline smalldata_tools/ana_funcs/droplet2Photons.py:85]: Should we indent the return statement back?

File: smalldata_tools/ana_funcs/droplet2Photons.py [hunk 2]
```diff
@@ -8,71 +8,256 @@
...
 
 class droplet2Photons(DetObjectFunc):
-    '''
-    return_img: whether or not to return the img with how many photons at each coordinate
-    threshold: # (noise sigma for) threshold
+    """
+    aduspphot: expected photon energy
+    offset: offset to convert number of photons into energy boundaries (defaults to 0.5*aduspphot)
+    photpts: energy boundaries for given number of photons (defaults to offset + N*aduspphot)
+    one_photon_info: energy range for a single photon ((defaults to [offset, offset+N*aduspphot])
```
Comment (vespos) [inline smalldata_tools/ana_funcs/droplet2Photons.py:19]: Duplicate argument with one_photon_limits

File: smalldata_tools/ana_funcs/droplet2Photons.py (hunk 1 above)
Comment (silkenelson) [reply smalldata_tools/ana_funcs/droplet2Photons.py]: Done

File: smalldata_tools/ana_funcs/droplet2Photons.py (hunk 2 above)
Comment (silkenelson) [reply smalldata_tools/ana_funcs/droplet2Photons.py]: argument was not duplicate, info line was

**Pull request merged**